backend/models/collaboration_request.py

```
@@ -0,0 +1,21 @@
+from sqlalchemy import Column, Integer, String, Text, ForeignKey, Enum
```

Ruff (0.8.2)

`sqlalchemy.String` imported but unused

Remove unused import: `sqlalchemy.String`

(F401)